tests/scripts/ci.py

```diff
@@ -394,7 +396,9 @@ def fn(
         # Add named test suites
         for option_name, (_, extra_scripts) in options.items():
             if kwargs.get(option_name, False):
-                scripts += extra_scripts
+                scripts.extend(
+                    script.format(name=name, build_dir=build_dir) for script in extra_scripts
```

Comment:
is `name` actually used anywhere?
```suggestion
                    script.format(build_dir=build_dir) for script in extra_scripts
```

Reply:
It currently isn't.  I had been thinking that the `task_config_build_{name}.sh` could be similarly handled in the formatting, but it didn't end up being necessary.  Updated as suggested, thank you!